Resources/Prototypes/_LostParadise/Research/diamond_production.yml

```
@@ -13,6 +13,18 @@
 
+- type: technology
+  id: LPPDiamondProcessing
+  name: lppresearch-technology-diamond-processing
+  icon:
+    sprite: _LostParadise/Structures/Machines/diamond_crusher.rsi
+    state: base
+  discipline: LPPDiamondProduction
+  tier: 1
+  cost: 2500
+  recipeUnlocks:
+  - LPPDiamondCrusherMachineCircuitboard
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)